config/views/check.py

```diff
@@ -1,5 +1,19 @@
-from .template_view_with_context import TemplateViewWithContext
+from django.http import JsonResponse
 
+from judgments.utils.utils import get_document_by_uri_from_cache
 
-class CheckView(TemplateViewWithContext):
-    template_name = "pages/check.html"
+
+def status(request):
+    document_cache_info = get_document_by_uri_from_cache.cache_info()
+
+    return JsonResponse(
+        {
+            "status": "ok",
```

Comment:
I'd be tempted to make this `OK` so `"OK"` appears in the output, in case that was being tested for.